### PR TITLE
Update refresh-stale.yml

### PR DESCRIPTION
**Pull request by victoriadrake**
- Don't refresh "blocked" issues
- Add "blocked" to exempt labels

### CODE REVIEW

File: .github/workflows/refresh-stale.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         stale-issue-message: 'Please comment if you are still working on this issue, as it has been inactive for 30 days. To give everyone a chance to contribute, we are releasing it to new contributors.'
         stale-issue-label: 'help wanted'
-        exempt-issue-labels: 'help wanted'
+        exempt-issue-labels: 'help wanted,blocked'
         stale-pr-message: 'Please comment if you are still working on this PR, as it has been inactive for 30 days. To give everyone a chance to contribute, we are releasing it for new contributors to take over.'
         stale-pr-label: 'help wanted'
         exempt-pr-labels: 'help wanted'
```
Comment (kingthorin) [inline .github/workflows/refresh-stale.yml:18]: Here too?

Comment (victoriadrake) [reply .github/workflows/refresh-stale.yml:18]: Good point

**Comment by kingthorin** [comment]
Think we could switch this to weekly? Are we actually concerned enough to need to check this daily?

**Comment by victoriadrake** [comment]
@kingthorin Sure, works for me.

**Pull request merged**